packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -372,8 +451,8 @@ The following parameters can be posted as a part of the URL-encoded form values
 
 | Parameter               | Description                                                                                                    | Type          |
 | :---------------------- | :------------------------------------------------------------------------------------------------------------- | :-----        |
-| token                   | An access token, ID token, or refresh token.                                                                   | String        |
-| token_type_hint         | Indicates the type of `token` being passed. Valid values are `access_token`, `id_token` and `refresh_token`.   | String (Enum) |
+| token                   | An access token, ID token, refresh token, or device secret<ApiLifecycle access="ea" />.                                                                   | String        |
+| token_type_hint         | Indicates the type of `token` being passed. Valid values: `access_token`, `id_token`, `refresh_token`, and `device_secret`<ApiLifecycle access="ea" />.   | String (Enum) |
 
 #### Response properties
```

Comment:
device_secret will have `sid`, do we also need to provide the example for it?

Reply:
I don't think the general OAuth2/OIDC api reference is the best place for it, rather have it in the guide.